package.json

```diff
@@ -34,19 +34,21 @@
     "npm": ">= 3.0.0"
```

Comment:
As well as specifying the NPM version, we could also add the supported Node.js version. Was there any issues with older NPM versions?

https://docs.npmjs.com/cli/v6/configuring-npm/package-json#engines

Reply:
I think this is a remnant of the original vue-gettext, I'll try to figure out what makes sense here.